### PR TITLE
Fix test suite on Windows

### PR DESCRIPTION
**Pull request by Turbo87**
This PR fixes most of the currently failing tests on Windows/AppVeyor. It sort of obsoletes #5470.

### CODE REVIEW

**Comment by Turbo87** [comment]
```
 Downloading PhantomJS 32 bit
   from 'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-windows.zip'
 Exception calling "GetResponse" with "0" argument(s): "The remote server return
 ed an error: (429) TOO MANY REQUESTS."
```

:bomb:

**Comment by Turbo87** [comment]
well, at least the node 5 build looks a lot better now :smiley:

**Comment by rwjblue** [comment]
Should this also delete the `.gitattributes` file?

**Comment by rwjblue** [comment]
Will need @kellyselden and/or @raytiley to review I think.  I'm definitely not a qualified Windows developer any more....

**Comment by trek** [comment]
Qualified© Windows® Developer

**Comment by kellyselden** [comment]
@rwjblue The motivation for the `.gitattributes` change was doing releases. Making sure the files on disk on a Windows machine were \n before sending to npm, no matter what the global git settings were on the machine. So even though adding the file got us into this mess, I think removing it would bring us back to square one.

**Comment by rwjblue** [comment]
Gotcha, makes sense

**Comment by Turbo87** [comment]
I agree with what @kellyselden said. I originally misunderstood the situation before the `.gitattributes` file was introduced and thought Windows users had always gotten `\r\n` files. But since the files published to NPM were always `\n` our test suite actually didn't test the reality. Even worse, if any Windows devs would clone the project without using auto CRLF conversion the test suite would have failed for them even before `.gitattributes` was introduced.

tl;dr having `.gitattributes` force consistent line endings on all platforms is a good thing and our tests should reflect that.

feel free to `r+` this PR. I'm not sure about the other failing tests, but I suspect that raising the timeouts on them might work. Since AppVeyor is actually really slow we could also skip the slow tests there and run them only on TravisCI. thoughts on that?

**Comment by rwjblue** [comment]
Does the .gitattributes file ensure that all newly added files have \n line endings? Do we need to do anything to ensure that?

**Comment by rwjblue** [comment]
@homu r+

**Comment by Turbo87** [comment]
@rwjblue I thought it did, but when I read about it yesterday it seemed to only concern `git checkout` but not `git commit`. I'm not sure if we need active protection against `\r\n` files, but if we do ESLint got us covered (http://eslint.org/docs/rules/linebreak-style.html)

**Comment by Turbo87** [comment]
@homu test failure looks like a flaky test again... :-/

**Comment by rwjblue** [comment]
Sounds good. Can you enable that rule, and rebase? Then we'll get this landed...

**Comment by Turbo87** [comment]
@rwjblue it's actually enabled already (https://github.com/ember-cli/ember-cli/blob/master/.eslintrc.js#L56-L57) :joy:

**Comment by rwjblue** [comment]
HA! Awesome...

**Comment by rwjblue** [comment]
ZOMG!  AppVeyor is green!

**Comment by kellyselden** [comment]
nice work @Turbo87 !
